app/services/onlinetesting/OnlineTestService.scala

```diff
@@ -85,10 +85,12 @@ trait OnlineTestService {
   }
 
   def registerAndInviteForTestGroup(application: OnlineTestApplication): Future[Unit] = {
-    val (invitationDate, expirationDate) = onlineTestDates
-    val registerAndInviteProcess = Future.sequence(getScheduleIdForApplication(application).map { scheduleId =>
-      registerAndInviteApplicant(application, scheduleId, invitationDate, expirationDate)
-    }).map { phase1Tests =>
+    val (invitationDate, expirationDate) = calcOnlineTestDates()
+
+    val registerAndInviteProcess = Future.sequence(
+      getScheduleIdsForApplication(application).map { scheduleId =>
```

Comment:
Because the getScheduleIdsForApplication returns multiple scheduleIds, and therefore we create multiple futures, and we need to wait for all the futures, so using Future.sequence turns them into a single future to map further operations